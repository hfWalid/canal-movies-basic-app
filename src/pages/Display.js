import React from 'react';

import {Display} from "../features/display";


const DisplayPage: function = (props) => {
    return (<Display {...props}/>);
};

export default DisplayPage;
