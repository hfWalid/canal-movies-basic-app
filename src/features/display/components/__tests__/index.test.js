import React from 'react';
import { cleanup, render } from 'react-testing-library';
import {Provider} from "react-redux";
import configureStore from "../../../../store";
import Display from "../index";

jest.mock('react-router-dom', () => ({
    useLocation: () => ({
        push: jest.fn(),
    }),
}));

// automatically unmount and cleanup DOM after the test is finished.
afterEach(cleanup);

describe('renders a single movie feature: display', () => {
    let store;
    beforeAll(() => {
        store = configureStore({});
    });

    test('renders display', async () => {
        const { asFragment } = render(
            <Provider store={store} >
                <Display />
            </Provider>
        );
        expect(asFragment()).toMatchSnapshot();
    });
});
