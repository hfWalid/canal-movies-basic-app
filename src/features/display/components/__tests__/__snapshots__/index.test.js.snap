// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a single movie feature: display renders display 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL QzUTY"
  />
</DocumentFragment>
`;
